### ISSUE TITLE
no javascript events

### ISSUE THREAD

**Issue by smcmurray** [adamjgrant/kickstart#188]
Using v3.x static js and css, the js doesn't work. Dropdown buttons don't dropdown, etc.

I use an AMD loader.

The documentation does not indicate that k$ has to be invoked manually. Is there some problem with load order?

**Comment by adamjgrant** [comment]
Yeah, the AMD loader could be the issue. Could you share some code?
